tests/Http/ClientTest.php

```diff
@@ -211,29 +210,16 @@ public function testClientHasDefaultUserAgent(): void
         $httpClient = $this->createHttpClientMock(200, '{}');
         $reqFactory = $this->createMock(RequestFactoryInterface::class);
         $requestStub = $this->createMock(RequestInterface::class);
-
-        /* @phpstan-ignore-next-line */
-        $requestStub->expects($this->any())
+        $requestStub->expects($this->exactly(2))
             ->method('withAddedHeader')
-            ->withConsecutive(
```

Comment:
Hi @stloyd I would like to know how we could ensure we have the correct user agent without this test?

Reply:
`willReturnCallback` should be used to test these

Reply:
Covered.